man/systemd.special.xml

```diff
@@ -512,7 +512,8 @@
           shall add <varname>Conflicts=</varname> dependencies to this
           unit for their service unit, which is implicitly done when
           <varname>DefaultDependencies=yes</varname> is set (the
-          default).</para>
+          default) along with <varname>Before=</varname> dependency.
+          </para>
```

Comment:
Should not it rather recommend the same, as systemd does implicitly - adding Before together with Conflicts? If there is only Conflicts, but no Before/After, then there is no ordering between two jobs and it may in principle happen that stopping of unit happens too late.